src/pip/_internal/basecommand.py

```diff
@@ -246,12 +242,12 @@ def main(self, args):
             return UNKNOWN_ERROR
         finally:
             # Check if we're using the latest version of pip available
-            if (not options.disable_pip_version_check and not
-                    getattr(options, "no_index", False)):
+            if not options.disable_pip_version_check and not getattr(
+                options, "no_index", False
+            ):
```

Comment:
I'm not 100% sure on the particular rationale for this one, I think it's trying to avoid line continuation characters when the line already has them built in as well as trying to be consistent (since function calls can have many more arguments, this formats every multi line function in a similar manner).